### ISSUE TITLE
Response Content-Type "text/html" does not match any acceptable types: image/x-xbitmap,image/x-ic...

### ISSUE THREAD

**Issue by loremattei** [wordpress-mobile/WordPress-iOS#14052]
**Sentry Url**: https://sentry.io/share/issue/5a600a7170bb436b966aab1a8592dde4/
**User Count**: 1102
**Count**: 10566
**First Release**: 26914
**First Seen**: 2020-04-09T18:38:57Z
**Last Seen**: 2020-05-05T09:36:43Z
**24 Hours**: 8189
**30 Days**: 10552

```
Response Content-Type "text/html" does not match any acceptable types: image/gif,image/tiff,image...

```

**Comment by loremattei** [comment]
This gets logged in Sentry, but it's not a crash, it's reported as an error, but it has an impressive amount of occurrences, so it's probably worth investigation.

**Comment by bummytime** [comment]
Fixed in #14058
